packages/ckeditor5-link/src/linkui.ts

```diff
@@ -373,18 +432,24 @@ export default class LinkUI extends Plugin {
 	}
 
 	/**
-	 * Creates a view for bookmarks.
+	 * Creates a view for links provider.
 	 */
-	private _createBookmarksView(): LinkBookmarksView {
+	private _createLinkProviderItemsView( provider: LinksProvider ): LinkProviderItemsView {
 		const editor = this.editor;
-		const view = new LinkBookmarksView( editor.locale );
+		const t = editor.locale.t;
+
+		const view = new LinkProviderItemsView( editor.locale );
+		const { emptyListPlaceholder, label } = provider;
+
+		view.set( 'emptyListPlaceholder', emptyListPlaceholder || t( 'No links available' ) );
+		view.set( 'headerLabel', label );
```

Comment:
```suggestion
		view.emptyListPlaceholder = emptyListPlaceholder || t( 'No links available' );
		view.headerLabel = label;
```
Only the first set is required as a method call to create a setter/getter and we usually do it in a constructor.